### PR TITLE
Maunope/network added firewall metrics

### PR DESCRIPTION
**Pull request by maunope**

### CODE REVIEW

File: blueprints/cloud-operations/network-dashboard/cloud-function/metrics/vpc_firewalls.py
Comment (aurelienlegrand) [inline blueprints/cloud-operations/network-dashboard/cloud-function/metrics/vpc_firewalls.py:91]: As discussed, I think there is an issue around this point to be troubleshooted.

**Review by aurelienlegrand — CHANGES_REQUESTED** [review]
Thanks Maurizio for the contribution!

I added a few comments, mostly naming and documentation related.
Also I tested the code and I am waiting for a fix before I merge this PR :)

File: blueprints/cloud-operations/network-dashboard/cloud-function/metrics/vpc_firewalls.py
Comment (maunope) [reply blueprints/cloud-operations/network-dashboard/cloud-function/metrics/vpc_firewalls.py:110]: k

File: blueprints/cloud-operations/network-dashboard/cloud-function/metrics.yaml
Comment (maunope) [reply blueprints/cloud-operations/network-dashboard/cloud-function/metrics.yaml:169]: k

File: blueprints/cloud-operations/network-dashboard/cloud-function/main.py
Comment (maunope) [reply blueprints/cloud-operations/network-dashboard/cloud-function/main.py:36]: y

File: blueprints/cloud-operations/network-dashboard/cloud-function/metrics/vpc_firewalls.py
Comment (maunope) [reply blueprints/cloud-operations/network-dashboard/cloud-function/metrics/vpc_firewalls.py]: fixed, you were right, it's firewalls_dict['project']

Comment (maunope) [reply blueprints/cloud-operations/network-dashboard/cloud-function/metrics/vpc_firewalls.py]: Fixed, you were right, it's firewalls_dict['project']

File: blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json
Comment (aurelienlegrand) [inline blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json:383]: Could you please add the ip_addresses_per_subnet_utilization metric in the dashboard that you removed?

File: blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json
Comment (aurelienlegrand) [inline blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json:420]: This should be firewalls_per_project_vpc_usage

File: blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json
Comment (aurelienlegrand) [inline blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json:441]: I think the metric should be "firewalls_per_project_vpc_usage":
                    "filter": "metric.type=\"custom.googleapis.com/firewalls_per_project_vpc_usage\" resource.type=\"global\""

File: blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json
Comment (maunope) [reply blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json]: y

File: blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json
Comment (maunope) [reply blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json]: y

File: blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json
Comment (aurelienlegrand) [inline blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json:512]: Could you please remove that line?

File: blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json
Comment (aurelienlegrand) [inline blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json:428]: Can you use the following instead (like the other charts):
"plotType": "LINE",

Comment (maunope) [reply blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json:428]: y

File: blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json
Comment (maunope) [reply blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json:512]: y

**Review by aurelienlegrand — APPROVED** [review]

**Comment by aurelienlegrand** [comment]
I think the PR is good now, thanks a lot for your contribution Maurizio!
LGTM

**Pull request merged**
